Source/WebCore/platform/graphics/ImageFrameWorkQueue.cpp

```diff
@@ -65,8 +65,9 @@ void ImageFrameWorkQueue::start()
     m_workQueue = WorkQueue::create("org.webkit.ImageDecoder", WorkQueue::QOS::Default);
 
     m_workQueue->dispatch([protectedThis = Ref { *this }, protectedWorkQueue = Ref { *m_workQueue }, protectedSource = this->protectedSource(), protectedDecoder = Ref { *decoder }] () mutable {
+        Ref protectedFrameRequestQueue { protectedThis->requestQueue() };
```

Comment:
Should not be this in the lambda capture above as the original code before the refactoring:

```
..., protectedRequestQueue = Ref { requestQueue() }, ...
```